features/step_definitions/redirect_steps.rb

```diff
@@ -13,3 +13,20 @@
 Then /^the redirect should send me to (.*)$/ do |url|
   expect(last_response.location).to eq url
 end
+
+When(/^an umapped old RubyForge URL is visited$/) do
+  visit 'http://some-arbitrary-unmapped-gem.rubyforge.org'
+end
+
+Then(/^a not found page should be presented$/) do
+  expect(Capybara.current_session.driver.response.status).to eq Rack::Utils.status_code(:not_found)
```

Comment:
I thought Capybara explicitly didn't expose response codes :-/ But it does! Fixed in 8535e71.